src/SlewLimiter.cpp

```diff
@@ -19,49 +21,74 @@ struct SlewLimiter : Module {
 		NUM_OUTPUTS
 	};
 
-	float out = 0.0;
+	float_4 out[4];
```

Comment:
Initialize your variables in their definitions.

Before release, I'd like you to go through your code and make sure that all variable definitions are initialized. When reading, it consumes several minutes to track down whether variables are initialized before they are used, so it would be easier to just initialize them in their definitions, such as `= {}`.

Reply:
Yup good point. A lot of these are inherited from the original polyphony PR, but I should have done a more thorough clean up. Will take note for future reference!

I've gone through everything again now, hopefully all in working order!